src/Utilities/TaskItem.cs

```diff
@@ -77,12 +77,28 @@ public TaskItem()
         /// </summary>
         /// <comments>Assumes the itemspec passed in is escaped.</comments>
         /// <param name="itemSpec">The item-spec string.</param>
+        public TaskItem(string itemSpec) => new TaskItem(itemSpec, treatAsFilePath: true);
+
+        /// <summary>
+        /// This constructor creates a new task item, given the item spec.
+        /// </summary>
+        /// <comments>
+        /// Assumes the itemspec passed in is escaped.
+        /// If <see name="treatAsFilePath" /> is set to <see langword="true" />, the value in <see name="itemSpec" />
+        /// will be fixed up as path by having backslashes replaced with slashes.
+        /// </comments>
+        /// <param name="itemSpec">The item-spec string.</param>
+        /// <param name="treatAsFilePath">
+        /// Specifies whether or not to treat the value in <see name="itemSpec" />
+        /// as a file path and attempt to normalize it.  Defaults to <see langword="true" />.
+        /// </param>
         public TaskItem(
-            string itemSpec)
+            string itemSpec,
+            bool treatAsFilePath = true)
```

Comment:
I *believe* combining 'coliding' explicit signature with and without default params is an antipattern.
Can we make the `treatAsFilePath` mandatory here?